cmd/podman/images/sign.go

```diff
@@ -48,6 +48,10 @@ func init() {
 	flags.StringVar(&signOptions.CertDir, certDirFlagName, "", "`Pathname` of a directory containing TLS certificates and keys")
 	_ = signCommand.RegisterFlagCompletionFunc(certDirFlagName, completion.AutocompleteDefault)
 	flags.BoolVarP(&signOptions.All, "all", "a", false, "Sign all the manifests of the multi-architecture image")
+
+	sigFileFlagName := "sigfile"
```

Comment:
It’s not very consistent to have the destination-affecting options named `--directory` and `--sigfile`. Why is `sig` in one and not the other? (Also, all other options separate words with dashes.)

Reply:
Ok going back to just using --authfile and dropping attempting to set the sigfile.
Since that was the original issue.